Material.Styles/CheckBox.xaml

```diff
@@ -19,22 +19,23 @@
                             <ColumnDefinition Width="Auto" />
                             <ColumnDefinition Width="*" />
                         </Grid.ColumnDefinitions>
-                        <Viewbox
-                            Width="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:SelectionControlAssist.Size)}"
-                            Height="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:SelectionControlAssist.Size)}"
-                            VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-                            ClipToBounds="False">
-                            <Grid>
-                                <Path x:Name="Graphic" Width="20" Height="20" Margin="-2 -2 0 0"/>
-                                <Ellipse x:Name="InteractionEllipse"
-                                         Fill="{TemplateBinding Background}" 
-                                         VerticalAlignment="Center" HorizontalAlignment="Center"
-                                         Width="0" Height="0" Opacity="0" Margin="-100"
-                                         ClipToBounds="False"
-                                         IsHitTestVisible="False">
-                                </Ellipse>
-                            </Grid>
-                        </Viewbox>
+                        <Grid>
+                          <Path x:Name="Graphic" Width="24" Height="24" Margin="0"/>
```

Comment:
This changes makes SelectionControlAssist.Size useless. Why?

Reply:
If SelectionControlAssist appear is purposal, I can return it back. (actually it's my mistakes too because I think without viewbox could more better)